.github/actions/spelling/patterns.txt

```diff
@@ -17,3 +17,10 @@ data:[a-zA-Z=;,/0-9+-]+
 El proyecto .* diferentes
 # Package family names
 \b[-.A-Za-z0-9]+_[a-z0-9]{13}\b
+# URLs
+(?:(?:(?:ht|f)tp(?:s?))\://)?(?:www.|[a-zA-Z].)[a-zA-Z0-9\-\.]+\.(?:com|edu|gov|mil|net|org|biz|info|name|museum|us|ca|uk)(?:\:[0-9]+)*(?:/(?:$|[a-zA-Z0-9\.\,\;\?\'\\\+&%\$#\=~_\-]+))*
```

Comment:
fwiw, often `http://` is a bug of `https://`.
And people regularly misspell domains and pieces of urls.

You're of course welcome to ignore both.

I regularly find typos in domains and paths, and regularly find places that are using `http:` when they really should use `https:`.

Reply:
Fair enough. If I want to allow this specific url (http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer) should I just add it to allow.txt, or create a pattern for it?

Reply:
For now, I've created a pattern for it

Reply:
allow only tolerates individual "words", so you could only add:
`rfc`
`gtm`
`corp`
`TSS`
`Http`
`Tsp`
`Server` (this should already be in the dictionary)

So, yes, if you have a specific sequence of tokens you have to use a pattern. -- Sorry it's a bit awkward. Again, still collecting feedback.

Reply:
Fwiw, here's the advice text from another project:

> * You can probably just add items into `.github/actions/spell-check/expect.txt`.
> * If you need to use a specific token in one place and it shouldn't generally be used, you can
> add an item in `.github/actions/spell-check/patterns.txt`.